tortoise/expressions.py

```diff
@@ -158,7 +147,7 @@ def resolve(self, resolve_context: ResolveContext) -> ResolveResult:
             except KeyError:
                 raise FieldError(
                     f"There is no non-virtual field {self.name} on Model {resolve_context.model.__name__}"
-                )
+                ) from None
```

Comment:
What is it for?

Reply:
Add `from None` will hide the KeyError message. Now that we should the FieldError, there is no need to show the previous KeyError info.

For Example:
- demo.py
```
class FieldError(Exception):
    pass

try:
    {}["a"]
except KeyError:
    raise FieldError("There is no non-virtual")
```
print:
```
Traceback (most recent call last):
  File "demo.py", line 6, in <module>
    {}["a"]
KeyError: 'a'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "demo.py", line 8, in <module>
    raise FieldError("There is no non-virtual")
__main__.FieldError: There is no non-virtual
```
- demo_with_from_none.py
```py
class FieldError(Exception):
    pass

try:
    {}["a"]
except KeyError:
    raise FieldError("There is no non-virtual") from None
```
print:
```
Traceback (most recent call last):
  File "demo_with_from_none.py", line 8, in <module>
    raise FieldError("There is no non-virtual") from None
__main__.FieldError: There is no non-virtual
```
See more at: https://docs.astral.sh/ruff/rules/raise-without-from-inside-except/

Reply:
Got it, thank you for the explanation!